src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -95,7 +95,7 @@ def create_project_for_organization(
 
             if add_org_default_team:
                 team = (
-                    Team.objects.filter(organization_id=organization_id)
+                    Team.objects.filter(organization_id=organization_id, status=TeamStatus.ACTIVE)
```

Comment:
This initially was a bug where we fetch the first team as being default team without looking at its status.